src/chttpd/src/chttpd_changes.erl

```diff
@@ -441,37 +462,43 @@ send_changes(Acc, Dir, FirstRound) ->
             fabric2_db:fold_changes(Db, StartSeq, DbEnumFun, Acc, Opts)
     end.
 
-
 can_optimize(true, {doc_ids, _Style, DocIds}) ->
-    MaxDocIds = config:get_integer("couchdb",
-        "changes_doc_ids_optimization_threshold", 100),
-    if length(DocIds) =< MaxDocIds ->
-        {true, fun send_changes_doc_ids/6};
-    true ->
-        false
+    MaxDocIds = config:get_integer(
+        "couchdb",
+        "changes_doc_ids_optimization_threshold",
+        100
+    ),
+    if
+        length(DocIds) =< MaxDocIds ->
+            {true, fun send_changes_doc_ids/6};
+        true ->
+            false
     end;
 can_optimize(true, {design_docs, _Style}) ->
     {true, fun send_changes_design_docs/6};
 can_optimize(_, _) ->
     false.
 
-
 send_changes_doc_ids(Db, StartSeq, Dir, Fun, Acc0, {doc_ids, _Style, DocIds}) ->
     Results = fabric2_db:get_full_doc_infos(Db, DocIds),
-    FullInfos = lists:foldl(fun
-        (#full_doc_info{}=FDI, Acc) -> [FDI | Acc];
-        (not_found, Acc) -> Acc
-    end, [], Results),
+    FullInfos = lists:foldl(
+        fun
+            (#full_doc_info{} = FDI, Acc) -> [FDI | Acc];
+            (not_found, Acc) -> Acc
+        end,
+        [],
+        Results
+    ),
     send_lookup_changes(FullInfos, StartSeq, Dir, Db, Fun, Acc0).
 
-
 send_changes_design_docs(Db, StartSeq, Dir, Fun, Acc0, {design_docs, _Style}) ->
     FoldFun = fun(FDI, Acc) ->
         case FDI of
             {row, Row} ->
                 DocId = proplists:get_value(id, Row),
                 {ok, [fabric2_db:get_full_doc_info(Db, DocId) | Acc]};
-            _ -> {ok, Acc}
+            _ ->
```

Comment:
+1. Even though putting short expressions on the same line is good. It is warranted only if all clauses have single expression in the body.

Reply:
+2